controllers/release/release_adapter_test.go

```diff
@@ -657,8 +657,11 @@ var _ = Describe("Release Adapter", Ordered, func() {
 		Expect(err).NotTo(HaveOccurred())
 		Expect(clusterName.String()).To(Equal("test-workspace"))
 
-		// Restore original targetContext value
+		// Restore original values
 		adapter.targetContext = adapter.context
+		patch = client.MergeFrom(releasePlan.DeepCopy())
+		releasePlan.Spec.Target.Workspace = ""
```

Comment:
I would store the value just before setting it in the previous patch. That way you can use a variable here and be certain you are restoring the original one.